atc/worker/worker.go

```diff
@@ -491,16 +492,19 @@ func (worker *gardenWorker) createVolumes(
 		return nil, err
 	}
 
-	streamedMounts, err := worker.cloneRemoteVolumes(
-		ctx,
-		logger,
-		spec.TeamID,
-		isPrivileged,
-		creatingContainer,
-		nonlocalInputs,
-	)
-	if err != nil {
-		return nil, err
+	streamedMounts := []VolumeMount{}
+	if len(nonlocalInputs) > 0 {
```

Comment:
@cirocosta Is this adding new logic aside from adding tracing to image streaming? I'm not sure what effect only cloning remote volumes when there are non local inputs has?